ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt

```diff
@@ -78,39 +76,29 @@ class MainActivity : ComponentActivity() {
             mService = null
         }
     }
+
     private fun copyFilesToExternalDir() {
         val am = resources.assets
-        var files: Array<String>? = null
-
-        try {
-            files = am.list("models/")
-        } catch (e: java.lang.Exception) {
-            Log.e(TAG, "#### Failed to get asset file list")
-            e.printStackTrace()
-            return
-        }
 
-        // Copy files into app-specific directory.
-        for (filename in files!!) {
-            try {
-                val inFile = am.open("models/$filename")
-                val outDir = getExternalFilesDir(null)!!.absolutePath
-                val outFile = File(outDir, filename)
-                val out: OutputStream = FileOutputStream(outFile)
-
-                val buffer = ByteArray(1024)
-                var read: Int
-                while ((inFile.read(buffer).also { read = it }) != -1) {
-                    out.write(buffer, 0, read)
+        am.list("models/")?.let { fileArray ->
+            val files: Array<String> = fileArray
```

Comment:
What happens if using fileArray directly instead of files?

Reply:
I accidentally created a meaningless variable.
Thank you :)